app/importers/unpaywall_publication_importer.rb

```diff
@@ -1,10 +1,20 @@
 # frozen_string_literal: true
 
 class UnpaywallPublicationImporter
+  def import_before(date: Time.now)
+    count = Publication.where(unpaywall_last_checked_at: nil..date).count
+    Rails.logger.info("Starting to import #{count} records from unpaywall")
+    Publication.where(unpaywall_last_checked_at: nil..date).or(Publication.where(unpaywall_last_checked_at: nil)).find_in_batches(batch_size: 500) do |publications|
+      publications.each do |p|
+        import_from_unpaywall(p)
+      end
+      Rails.logger.info('Processed 500 Records')
+    end
+  end
+
   def import_all
```

Comment:
We should delete this, and make sure it's not being used anywhere else.

Reply:
roger!